resources/postgres/postgres.go

```diff
@@ -42,9 +42,11 @@ func (i *Instance) Start(port int, stdout, stderr io.Writer, waitSec int) (err e
 
 	i.port = port
 
+	// See -p -k -h option definitions in the link below.
+	// https://www.postgresql.org/docs/current/app-postgres.html
```

Comment:
let's explicitly set the socket directory to GetPostgresSocketDir

Reply:
Done.